### DISCUSSION TITLE
Meta-flutter rebase from dunfell to kirkstone caused build breakage.

### DISCUSSION THREAD

**Discussion by Anantha007** [meta-flutter/meta-flutter#313] [General]
I am using the meta-flutter repo for about past 10 months without any issue, I was pointing to dunfell revision. But with latest commit made from aug 4, 2023, we're getting below errors. 
It is pointing errors with respect to user id & container ids. 
Can someone help me out with this? for now, I changed the revision to pass through our builds.

**Comment by jwinarske** [comment]
@Anantha007 what are the errors you are seeing?  Also it may require changing some recipe names.  Bbclass usage has been mostly removed.  It will now install multiple engine variants just by adding `flutter-engine`.  The runtime types for flutter-engine are now controlled by package config variables, which default to debug, profile, and release.  It will skip installing apps for debug by default, and there is a variable for that.  The default bundle folder path has changed as a result.  The other is I factored out the requirements for meta-ckand, and meta-rust to dynamic layers.  That's most of the breaking changes.  I'll update the README.  Other than that what are you seeing?
